### PR TITLE
Add nav dropdown and components for about and team pages

### PR DESCRIPTION
**Pull request by khalilcodes**
### Summary

This PR adds the collaborators section in the frontoage, and adds about and teams pages with dropdown links for both in nav similar to current lifeitself.org site.

### Changes

* Add components `collaborators.jsx`, `Team.jsx`
* Add all image assets for teams and collaborators in  their respective folders - `content/assets/site/team` and `content/assets/site/collaborators`
* New pages @ [about](https://deploy-preview-189--zesty-liger-3a5c29.netlify.app//about) and [people](https://deploy-preview-189--zesty-liger-3a5c29.netlify.app//people)
* Dropdown nav for about in `contig.js`

### CODE REVIEW

File: site/content/config.js [hunk 1]
```diff
@@ -9,7 +9,10 @@ const userConfig = {
     text: "Life Itself",
   },
   navLinks: [
-    { href: "/about", name: "About" },
+    { name: "About", subItems: [
```
Comment (rufuspollock) [inline site/content/config.js:12]: I really don't want dropdowns atm - it's bad UX i think (plus the UI functionality here is a bit broken - which i will flag)

File: site/content/components/collaborators.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+export default function Collaborators() {
+  return (
```
Comment (rufuspollock) [inline site/content/components/collaborators.jsx:2]: I feel we probably want to implement this stuff as loop over an array of collaborators e.g.

const collaborators = [
  {
	title:
	url: 
        image: 
  }
]

...

<section>
  ....
  for collaborator in collaborators ...
```

collaborators could even be loaded from a data file ... (but that's the future)

**Review by rufuspollock — CHANGES_REQUESTED** [review]
Generally looks good. One comment re making collaborators structure similar to how you have done team stuff in terms of having a list. Also suggest removing drop-down for now (don't want dropdowns)

File: site/content/config.js (hunk 1 above)
Comment (khalilcodes) [reply site/content/config.js]: Fixed 👍

File: site/content/components/collaborators.jsx (hunk 2 above)
Comment (khalilcodes) [reply site/content/components/collaborators.jsx:40]: This is a list now.

**Comment by rufuspollock** [comment]
@khalilcodes looks great. merging.

**Pull request merged**